packages/@pollyjs/adapter/src/index.js

```diff
@@ -144,7 +144,19 @@ export default class Adapter {
       await this.timeout(pollyRequest, recordingEntry);
       pollyRequest.action = ACTIONS.REPLAY;
 
-      return this.onReplay(pollyRequest, recordingEntry);
+      const { status, statusText, headers, content } = recordingEntry.response;
+      const normalizedResponse = {
```

Comment:
Create a normalized response for all adapter to use. It would be nice if we could use the Response class here from core.